pom.xml

```diff
@@ -86,6 +87,40 @@
         </pluginRepository>
     </pluginRepositories>
 
+
+    <!-- pin transitive dependencies for CVEs -->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
```

Comment:
I couldn't find httpclient, jose4j dependencies in the dependency tree. Can we exclude those from declaring here? Thanks!